entries/w/whalefin.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Whalefin": {
+    "domain": "whalefin.com",
+    "additional-domains": ["ambergroup.io"],
+    "url": "https://www.whalefin.com",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
```

Comment:
The documentation only mentions Google Authenticator, i.e. `totp`. Do you have further information about 2FA via `email` and `sms`?

Reply:
thanks for reviewing, and yes - there are a few pages on sms/email 2FA for sensitive actions
https://support.whalefin.com/hc/en-us/articles/4403359729945
https://support.whalefin.com/hc/en-us/articles/4409939331737
https://support.whalefin.com/hc/en-us/articles/4409952407321

Reply:
These "sensitive actions" do not seem to include logging in, so it's _authorization_ rather than _authentication_.

Reply:
2FA for login can also be regarded as authentication tho, depending on how you see and use it in different scenarios, the 2FA through sms/email is only required when taking actions that may cause losing money.

Reply:
That's what I mean - `sms` and `email` are only used for _authorization_; `totp` can stay.

Reply:
done